pkgs/desktops/enlightenment/efl.nix

```diff
@@ -90,30 +94,25 @@ stdenv.mkDerivation rec {
     xorg.xcbutilkeysyms
   ];
 
-  # ac_ct_CXX must be set to random value, because then it skips some magic which does alternative searching for g++
-  configureFlags = [
-    "--enable-sdl"
-    "--enable-drm"
-    "--enable-elput"
-    "--with-opengl=full"
-    "--enable-image-loader-jp2k"
-    "--enable-xinput22"
-    "--enable-multisense"
-    "--enable-liblz4"
-    "--enable-systemd"
-    "--enable-image-loader-webp"
-    "--enable-harfbuzz"
-    "--enable-xine"
-    "--enable-fb"
-    "--disable-tslib"
-    "--with-systemdunitdir=$out/systemd/user"
-    "ac_ct_CXX=foo"
+  mesonFlags = [
```

Comment:
I guess the other flags corresponding to former `configure` flags are ON by default or auto-detected?
Things like `harfbuzz`, `systemd`, `elput` ..

Reply:
Except `--enable-fb`, which defaults to false.

Reply:
[`meson-options.txt`](https://git.enlightenment.org/core/efl.git/tree/meson_options.txt) does not mention ` --enable-elput` abd `--enable-multisense`. According to raster (on IRC) they do not exist anymore and  they are on by default, or they are implied by some other config option.

Reply:
:+1: thanks for the research